src/components/dashboard/navigation.js

```diff
@@ -56,9 +215,10 @@ class Navigation extends Component {
             </NavLink>
 
             <NavLink
-              className="NavLink  hidden"
```

Comment:
Can you add these hidden back, we're not currently using these routes for anything.